frontend/app/app.css

```
@@ -75,6 +75,20 @@ body {
 
+@keyframes swap-glow {
+  0%,
+  100% {
+    box-shadow: 0 0 10px 2px rgba(249, 115, 22, 0.25);
+  }
+  50% {
+    box-shadow: 0 0 22px 6px rgba(249, 115, 22, 0.55);
+  }
+}
+
+@utility animate-swap-glow {
```

Stylelint (17.5.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)